src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -156,9 +162,34 @@
   <data name="ExactArgumentDescription" xml:space="preserve">
     <value>Find app using exact match</value>
   </data>
+  <data name="ExperimentalArgumentDescription" xml:space="preserve">
+    <value>Experimental argument for demonstration purposes</value>
+  </data>
+  <data name="ExperimentalCommandLongDescription" xml:space="preserve">
+    <value>This command is an example on how to implement an experimental feature. To turn on go to winget settings and enable experimentalCmd or experimentalArg features.</value>
+  </data>
+  <data name="ExperimentalCommandShortDescription" xml:space="preserve">
+    <value>Experimental feature example</value>
+  </data>
+  <data name="ExperimentalFeatureName" xml:space="preserve">
+    <value>Experimental feature:</value>
+  </data>
   <data name="ExtraPositionalError" xml:space="preserve">
     <value>Found a positional argument when none was expected</value>
   </data>
+  <data name="FeatureDisabledMessage" xml:space="preserve">
+    <value>If you want to try this feature, please go to winget settings and enable it. See</value>
+  </data>
+  <data name="FeaturesCommandLongDescription" xml:space="preserve">
+    <value>Shows the status of experimental features. Experimental features can be turn on via winget settings.</value>

```

Comment:
>winget settings [](start = 95, length = 15)

You will have to put a "lock" on this, see this elsewhere in the file:
    <comment>{Locked="VirtualTerminal"}</comment>